eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/io/JsonNodeToClusteredVariantProcessor.java

```diff
@@ -32,74 +31,70 @@
     }
 
     private ClusteredVariant parseJsonNodeToClusteredVariant(JsonNode jsonRootNode) {
-        String assemblyAccession, contig;
-        assemblyAccession = contig = DEFAULT_VALUE_STRING;
-        // @see https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=9606
+        // @see <a href=https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=9606>Human Tax ID.</a>
         int taxonomyAccession = 9606;
-        long start = 0L;
 
         // JSON date in ISO-8601 format
         DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-d'T'HH:mm'Z'");
         LocalDateTime createdDate = LocalDateTime.parse(jsonRootNode.path("create_date").asText(), formatter);
+
         VariantType type = prepareVariantType(jsonRootNode.path("primary_snapshot_data").path("variant_type").asText());
+
         JsonNode infoNode = jsonRootNode.path("primary_snapshot_data").path("placements_with_allele");
         for(JsonNode alleleInfo : infoNode) {
+
             boolean isPtlp = alleleInfo.path("is_ptlp").asBoolean();
             JsonNode assemblyInfo = alleleInfo.path("placement_annot").path("seq_id_traits_by_assembly");
+            JsonNode spdi = alleleInfo.path("alleles").get(0).path("allele").path("spdi");
             if(assemblyInfo.size() == 0 || !isPtlp) {
                 continue;
             }
-            assemblyAccession = assemblyInfo.get(0).path("assembly_accession").asText();
-            String alternateAllele, referenceAllele;
-            for (JsonNode alleleAndHgvs : alleleInfo.path("alleles")) {
-                JsonNode spdi = alleleAndHgvs.path("allele").path("spdi");
-                alternateAllele = spdi.path("inserted_sequence").asText();
-                referenceAllele = spdi.path("deleted_sequence").asText();
-                if(alternateAllele.equals(referenceAllele)) {
-                    contig = spdi.path("seq_id").asText();
-                    start = spdi.path("position").asLong(); //dbSNP stores in 0 base, EVA in 1 base
-                    return new ClusteredVariant(assemblyAccession, taxonomyAccession, contig, start,
-                            type, Boolean.FALSE, createdDate);
-                }
-            }
-        }
-        // no PLTP found
-        logger.error("No PLTP found for accession {}", jsonRootNode.path("refsnp_id").asLong());
-        return new ClusteredVariant(assemblyAccession, taxonomyAccession, contig, start,
+
+            String assemblyAccession = assemblyInfo.get(0).path("assembly_accession").asText();
+            String contig = spdi.path("seq_id").asText();
+            // DbSNP JSON in 0 base, EVA in 1 base
+            // @see <a href=https://api.ncbi.nlm.nih.gov/variation/v0/>DbSNP JSON 2.0 schema specification</a>
+            long start = spdi.path("position").asLong() + 1;
+
+            return new ClusteredVariant(assemblyAccession, taxonomyAccession, contig, start,
                 type, Boolean.FALSE, createdDate);
```

Comment:
Align the indentation with the first bracket to follow our coding style guidelines. Same for the call to `logger.error` below.